src/popup/router/pages/FungibleTokens/TokenDetails.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <div class="token-details">
+    <div class="token-header">
+      <div class="token-profile">
+        <Avatar
+          v-if="tokenData.contract || tokenData.image"
+          :address="tokenData.contract"
+          :src="tokenData.image || null"
+          size="xlg"
+        />
+        <div class="amount">
+          <span class="text-ellipsis max-space" :title="tokenData.convertedBalance || '0.00'">{{
+            tokenData.convertedBalance || '0.00'
+          }}</span>
+          <span class="symbol text-ellipsis max-space" :title="tokenData.symbol">{{
+            tokenData.symbol
+          }}</span>
+          <FormatFiatCurrency
+            class="text-ellipsis max-space"
+            :balance="tokenData.balanceCurrency"
+          />
+        </div>
+      </div>
+      <div class="token-actions">
+        <Button bold>
+          <RouterLink :to="{ name: 'send' }">{{ $t('pages.token-details.send') }}</RouterLink>
+        </Button>
```

Comment:
```suggestion
        <Button bold :to="{ name: 'send' }">
          {{ $t('pages.token-details.send') }}
        </Button>
```
This would make the whole button clickable, not only the center 🤦